packages/edit-site/src/components/style-book/index.js

```diff
@@ -182,6 +184,31 @@ function useMultiOriginPalettes() {
 	return palettes;
 }
 
+/**
+ * Get deduped examples for single page stylebook.
+ * @param {Array} examples Array of examples.
+ * @return {Array} Deduped examples.
+ */
+export function getExamplesForSinglePageUse( examples ) {
+	const examplesForSinglePageUse = [];
+	const overviewCategoryExamples = getExamplesByCategory(
+		{ slug: 'overview' },
+		examples
+	);
```

Comment:
In the StyleBook page for a classic theme, we would expect colors, gradients, and duotone palettes to be displayed, unlike in a block theme.

But because the `"overview"` slug is specified here, I think gradients and duotone palettes are excluded. Perhaps the logic of this function needs to be tweaked.

Reply:
The gradients and duotones still appear, just lower down, beneath "Headings". This is also how we're showing things in the non-tabbed stylebook for block themes currently. 

The idea for block themes is to use the global styles UI to navigate between different sections. Here, not having that UI the experience is a bit different. I think as a first iteration this is OK, but later on we should consider what kind of navigation we can provide for this stylebook when there's no global styles UI available.